pkg/platform/kube/platform.go

```diff
@@ -621,6 +621,10 @@ func (p *Platform) GetExternalIPAddresses() ([]string, error) {
 	return nil, errors.New("No external addresses found")
 }
 
+func (p *Platform) GetDefaultHTTPIngressHostTemplate() string {
+	return p.Platform.GetDefaultHTTPIngressHostTemplate()
```

Comment:
If it's the abstract implementation, I don't think this and the `local/platform.go` wrappers are needed